insights/specs/datasources/ssl_certificate.py

```diff
@@ -102,3 +104,31 @@ def mssql_tls_cert_file(broker):
     if mssql_conf_content.has_option("network", "tlscert"):
         return mssql_conf_content.get("network", "tlscert")
     raise SkipComponent
+
+
+@datasource(RsyslogAllConf, HostContext)
+def rsyslog_tls_cert_file(broker):
+    """
+    Get the rsyslog tls certificate file path configured by "DefaultNetstreamDriverCertFile"
+
+    Arguments:
+        broker: the broker object for the current session
+    Returns:
+        str: Returns the SSL certificate file path configured by "DefaultNetstreamDriverCertFile"
+    Raises:
+        SkipComponent: Raised if "DefaultNetstreamDriverCertFile" isn't found
+    """
+    rsyslog_objs = broker[RsyslogAllConf]
+    for obj in rsyslog_objs.values():
+        for item in obj:
+            if 'DefaultNetstreamDriverCertFile' in item:
+                # basic format
+                if '$DefaultNetstreamDriverCertFile' in item:
+                    return item.split()[-1].strip()
+                else:
+                    # advanced format, it is set in global block, and the global line contains all the content in it
```

Comment:
IIUC, this `if` and `else` are describing two parallel cases: `basic format` or `advanced format` . If this is the case, it is better to have the two comment lines with the same indent.